setup.py

```diff
@@ -4,7 +4,7 @@
     simulator"
 
 with open("README.md", "r") as readme_file:
```

Comment:
```suggestion
with open("README.md", "r", encoding="utf-8") as readme_file:
```
encoding is `open()` argument afaik